lighthouse-core/config/config.js

```diff
@@ -381,9 +381,13 @@ class Config {
    * @param {!Array<string>=} skipAuditIds
    * @return {!Object<string, {audits: !Array<{id: string}>}>}
    */
-  static filterCategoriesAndAudits(oldCategories, categoryIds = [], auditIds = [],
-      skipAuditIds = []) {
+  static filterCategoriesAndAudits(oldCategories, categoryIds, auditIds, skipAuditIds) {
```

Comment:
True. Not sure what the MO is for closure docs. IIRC, if you don't specify a default arg in the arg list, the var needs to be prefixed with `opt_`. That might be outdated though. cc @brendankenny